packages/website/components/account/addPaymentMethodForm/addPaymentMethodForm.js

```diff
@@ -49,7 +50,7 @@ const AddPaymentMethodForm = ({ setHasPaymentMethods, setEditingPaymentMethod, c
         if (error) throw new Error(error.message);
         if (!paymentMethod?.id) return;
         const currStorageSubscription = planIdToStorageSubscription(currentPlan);
-        await userBillingSettings(paymentMethod.id, currStorageSubscription);
+        await userBillingSettings(constants.TERMS_OF_SERVICE_VERSION, paymentMethod.id, currStorageSubscription);
```

Comment:
hmm. Did the user definintely consent to this TOS at this point? e.g. if the end-user is just adding their credit card (in the lower left of the /account/payment page) but not necessarily choosing a plan and checking the agreement checkbox, we probably don't want to record that they consented to anything.

Realizing this made me revisit this other comment above https://github.com/web3-storage/web3.storage/pull/2028/files#r998202155

Reply:
fixed via https://github.com/web3-storage/web3.storage/pull/2028/commits/eb506e338c403f970ce0bb6c19fcc35d92ea04ec